docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -28,42 +28,57 @@ Model Builder helps you create models for the following machine learning scenari
 - Issue classification (multiclass classification): Classify textual data into 3 or more categories.
 - Price prediction (regression): Predict a numeric value.
 - Image classification (deep learning): Categorize images based on characteristics.
+- Object detection (deep learning): Find objects within images. This can find one or more objects and label them accordingly.
 - Custom scenario: Build custom scenarios from your data using regression, classification, and other tasks.
 
-This article covers classification and regression scenarios with textual or numerical data, and image classification scenarios.
+This article covers classification and regression scenarios with textual or numerical data, image classification, and object detection scenarios.
 
 ## Load text or numeric data from a file
 
 You can load text or numeric data from a file into Model Builder. It accepts comma-delimited (CSV) or tab-delimited (TSV) file formats.
```

Comment:
In terms of file extensions, it supports CSV, TSV, and TXT. For delimiters, CSV,TSV and semicolons are accepted.